README.md

```diff
@@ -117,3 +117,24 @@ changed you can use the following command in that application:
 
 The above commands are very simple, however there are many options as far as dictating which versions are
 installed that can be used. For more information see [the official Composer documentation](https://getcomposer.org/doc/articles/versions.md#writing-version-constraints).
+
+## Install Composer Dependencies
+
+```
+./bin/composer install
+```
+
+## Code Formatting Standards
+#### Run PHP_CodeSniffer to see the existing styling issues.
+```
+composer sniff
```

Comment:
Do these commands need the `./bin/` prefix?

Reply:
They do not as these script commands are defined in the `composer.json` file. 

That does raise a question of consistency though. We have some scripts that live in `bin` and can be ran from there. We have other scripts that are expected to be ran from the composer command. It'd be nice to settle on one way.

Reply:
We should neither expect nor require people to have composer installed locally.  I do not and do not wish to.  That's why I added the composer docker script to the PHP Codesniffer repo. I think we should create something similar here and update the readme appropriately.

Reply:
@drew-acheson This should be resolved in the latest update. Give me a shout if it gives you any issues.

Reply:
Works great, thanks Tim!